### PR TITLE
chore(util): update TestListFilesInDir test

### PR DESCRIPTION
**Pull request by b00f**
## Description

This PR updates the `TestListFilesInDir` test and add directory to the test.

### CODE REVIEW

**Review by Ja7ad — APPROVED** [review]

File: util/io_test.go [hunk 1]
```diff
@@ -217,16 +217,23 @@ func TestSanitizeArchivePath(t *testing.T) {
 func TestListFilesInDir(t *testing.T) {
 	tmpDir := TempDirPath()
 
-	file1, err := os.Create(filepath.Join(tmpDir, ".file1"))
+	file1Path := filepath.Join(tmpDir, ".public_file")
```
Comment (b00f) [inline util/io_test.go:220]: Opps, we should not have dot here. @Ja7ad please review carefully. Let's fix this test again

Comment (Ja7ad) [reply util/io_test.go:220]: revert it?

Comment (b00f) [reply util/io_test.go:220]: No, new PR. Only remove dot from public file

Comment (Ja7ad) [reply util/io_test.go:220]: https://github.com/pactus-project/pactus/pull/1566

**Pull request merged**